cli/flox-rust-sdk/src/providers/publish.rs

```diff
@@ -155,8 +219,13 @@ where
             rev: self.env_metadata.build_repo_ref.rev.clone(),
             rev_count: self.env_metadata.build_repo_ref.rev_count as i64,
             rev_date: self.env_metadata.build_repo_ref.rev_date,
-            cache_uri: None,
+            cache_uri: self.cache.map(|c| c.cache_uri().to_string()),
```

Comment:
suggestion: make `cache_uri` an `Option<Url>`?
suggestion: `cache_uri` -> `cache_url` (L), for the same reason as above